packages/editor/src/editor/sidebar/fields/InputFieldWithBrowser.tsx

```diff
@@ -15,22 +15,37 @@ export const InputFieldWithBrowser = ({
   options
 }: InputFieldProps & { browsers: Browser[]; options?: TextBrowserFieldOptions }) => {
   const [open, setOpen] = useState(false);
+  const inputRef = useRef<HTMLInputElement>(null);
   const attrBrowser = useAttributeBrowser(options?.onlyAttributes, options?.onlyTypesOf);
   const logicBrowser = useLogicBrowser();
 
   const activeBrowsers = [...(browsers.includes('ATTRIBUTE') ? [attrBrowser] : []), ...(browsers.includes('LOGIC') ? [logicBrowser] : [])];
 
+  const customFocusOnDialogClose = (e: Event) => {
```

Comment:
I think this is already much slimmer and clearer, but I'm not sure whether I should move this function into a utility class or keep it completely separate/independent within the component. What do you think? If I should move it to its own utility, are there any existing utilities where it might fit?

Reply:
maybe withing the utils folder i could make a new file "focus.ts" and adjust the customFocusOnDialogClose as follow:
`const customInputFocusOnDialogClose = (e: Event, value: string, input: HTMLInputElement | null) => {`